portfolio/src/main/webapp/tasksApi.js

```diff
@@ -0,0 +1,203 @@
+var GoogleAuth;
+
+// Scope for read access to Tasks API
+const SCOPE_TASKS_READ = 'https://www.googleapis.com/auth/tasks.readonly';
+
+const TASKS_CLIENT_ID =
+    '499747085593-hvi6n4kdrbbfvcuo1c9a9tu9oaf62cr2.apps.googleusercontent.com';
+
+const TASKS_DISCOVERY_DOCS =
+    'https://www.googleapis.com/discovery/v1/apis/tasks/v1/rest';
+
+var TASKS_API_KEY = '';
+
+function fetchApiKey() {
+  fetch('./appConfigServlet')
+      .then(response => response.json())
+      .then((responseJson) => {
+        TASKS_API_KEY = responseJson['API_KEY'];
+      });
+}
+
+function handleClientLoadTasks() {
+  // Load the API's client and auth2 modules.
+  // Call the initClient function after the modules load.
+  gapi.load('client:auth2', initClientTasks);
+}
+
+function initClientTasks() {
+  fetchApiKey();
+
+  // Initialize the gapi.client object, which app uses to make API requests.
+  // Get API key and client ID from API Console.
+  // 'scope' field specifies space-delimited list of access scopes.
+  gapi.client
+      .init({
+        'apiKey': TASKS_API_KEY,
```

Comment:
You don't need the single quotes in keys.

Reply:
Done.